hazelcast/src/main/java/com/hazelcast/config/CardinalityEstimatorConfig.java

```diff
@@ -153,8 +146,14 @@ public String toString() {
                 + asyncBackupCount + ", readOnly=" + readOnly + '}';
     }
 
-    private static class CardinalityEstimatorConfigReadOnly
-            extends CardinalityEstimatorConfig {
+    CardinalityEstimatorConfigReadOnly getAsReadOnly() {
```

Comment:
Those methods are used in tests for `UnsupportedOperationException` in the read-only configs, so none of them can be private.